adoc/chapters/programming_interface.adoc

```diff
@@ -9074,6 +9074,16 @@ template<access::address_space ASP, access::decorated IsDecorated>
 
 Move the value of the left hand side [code]#multi_ptr# into the [code]#generic_ptr#.
 
+a@
+[source]
+----
+template <typename value_type, access::address_space Space, access::decorated DecorateAddress>
+    reference operator[](std::ptrdiff_t i) const
```

Comment:
Why is the operator shown as a template function here?  It's not a template in the code synopsis.

Reply:
For consistency with other declarations.

I do agree however the spec needs a clean up on that point, it doesn't quite make sense

Reply:
I see.  Yes, all the `multi_ptr` operators should be cleaned up.  It's probably better to do that in a separate PR.

Reply:
Yes, but at least remove the template here so we can merge this first.

Reply:
sure, done

Reply:
> Yes, all the multi_ptr operators should be cleaned up. It's probably better to do that in a separate PR.

Opened one for this https://github.com/KhronosGroup/SYCL-Docs/pull/154